### ISSUE TITLE
Command-line API

### ISSUE THREAD

**Issue by Daeloth** [delthas/autopunch#3]
I'm trying to apply some patches to a game using a single .bat, but I haven't been able to add autopunch to it. Could I get a process flag?

**Comment by delthas** [comment]
Makes sense. Sometimes autopunch will have to update itself and will probably need to show a UI to do so. Is that OK?

**Comment by Daeloth** [comment]
Yes, no problem with that

**Comment by M4RCK5** [comment]
how can do the injection process through cli??
